ci-operator/config/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-main.yaml

```diff
@@ -83,6 +83,16 @@ tests:
   container:
     from: test-bin
   run_if_changed: (\.md|^Makefile|^hack/.*markdown.*|^.markdown.*)$
+- as: ibu-seed-generation
```

Comment:
s/ibu/ibx?

Reply:
afaik it's not limited so I'll update it to say ibx